sec_certs/helpers.py

```diff
@@ -198,15 +198,20 @@ def repair_pdf(file: Path) -> None:
     pdf.save(file)
 
 
-def convert_pdf_file(pdf_path: Path, txt_path: Path, options) -> str:
-    response = subprocess.run(
-        ["pdftotext", *options, pdf_path, txt_path], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, timeout=60
-    ).returncode
-    if response == 0:
-        return constants.RETURNCODE_OK
-    else:
+def convert_pdf_file(pdf_path: Path, txt_path: Path) -> str:
+    try:
+        with pdf_path.open("rb") as pdf_handle:
+            pdf = pdftotext.PDF(pdf_handle, "", True)
```

Comment:
Hopefully this is OK to be executed from the celery worker (and won't bring back issues addressed in #135 and #141). I can test it when the branch matures (ideally before merge).